### DISCUSSION TITLE
Please add ability to force SSO on a per-user basis

### DISCUSSION THREAD

**Discussion by nneul** [community/community#11814] [Enterprise]
We're using GH Enterprise Cloud, and several of our sub-organizations have saml sso fully mandated, but not all. It would make things a LOT easier if we could gradually move to required SSO on a per user basis.

Please consider either:

A) Allow org owner to exempt an org member from SSO requirement (No, external collaborator is NOT a workaround for this, that eliminates ability to be a member of a team, and requires every single repo invite to be individually accepted.)
   or
B) Allow org owner to set "require SSO" on a per user basis. 
   or (best)
C) Every user gets an org-owner-only setting: "SSO Required: Inherit from org (default), Required, Not Required". I would suggest the same functionality for the MFA requirement. The org and enterprise should still have the ability to say 'forced globally with no overrides'.

**Comment by aaron-0** [comment]
Allowing external collaborators to be members of teams would also meet my needs.
